src/extensibility/ExtensionManagerDialog.js

```diff
@@ -311,10 +322,12 @@ define(function (require, exports, module) {
         $searchClear = $(".search-clear", $dlg);
 
         function setActiveTab($tab) {
-            models[_activeTabIndex].scrollPos = $(".modal-body", $dlg).scrollTop();
+            if (models[_activeTabIndex]) {
+                models[_activeTabIndex].scrollPos = $(".modal-body", $dlg).scrollTop();
+            }
             $tab.tab("show");
-            $(".modal-body", $dlg).scrollTop(models[_activeTabIndex].scrollPos || 0);
-            $searchClear.click();
+            $(".modal-body", $dlg).scrollTop((models[_activeTabIndex] && models[_activeTabIndex].scrollPos) || 0);
+            clearSearch();
```

Comment:
When I tested this, clearsearch() also threw an exception during the call to updateSearchDisabled(). I think this statement needs to be also checked for `models[_activeTabIndex]` to exist. Maybe you want to wrap both the statements in an if statement